src/utils/star-atlas.ts

```diff
@@ -37,25 +37,26 @@ export const getStarAtlasData = async (cluster: Cluster) => {
     {}
   );
 
-  const markets = starAtlasApiResponse.data.reduce<SerumMarketInfoMap>(
-    (map, { markets, symbol }) => {
+  const markets = starAtlasApiResponse.data.reduce<ISerumMarketInfo[]>(
```

Comment:
wont this require a front end change to consume an array over a map?

Reply:
We query the Star Atlas API separately on the frontend. This is specifically used for pricing data.